app/invites/client/admin/views.js

```diff
@@ -0,0 +1,2 @@
+import './adminInvites.html';
```

Comment:
You can import the html file on top of the js file and then import the js file directly on `app/invites/client/admin/route.js` making this file unecessary